### ISSUE TITLE
All font variables in css should specify a fallback

### ISSUE THREAD

**Issue by petemill** [brave/leo#482]
Fallback should at least be sans-serif

**Comment by AlanBreck** [comment]
@aguscruiz, can you define fallback fonts in Figma which can be included in the export?

**Comment by aguscruiz** [comment]
> @aguscruiz, can you define fallback fonts in Figma which can be included in the export?

No, sadly I can't define that in Figma. But if it helps, we could use this 
https://systemfontstack.com/

**Comment by AlanBreck** [comment]
My larger concern is that since it's not configurable in Figma, and therefore included in the export, we'll have to hard code that the fallback is `sans-serif`, so if we ever introduced `serif` (like with Fletcha for marketing), we'd have to hardcode a mapping for the fonts to their fallbacks.

Not a big deal, especially given the limited number of fonts we use. Just not ideal.
